src/calculateRentalCost.js

```diff
@@ -4,7 +4,20 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const price = 40;
+  const discount20 = 20;
+  const discount50 = 50;
+  const basePrice = days * price;
+
+  if (days >= 7) {
```

Comment:
Consider using variables instead of magic numbers

Reply:
Check the checklist for better naming https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md